### PR TITLE
Added files  to b_1 with HW_JS_1

### PR DESCRIPTION
**Pull request by Labrosimova**

### CODE REVIEW

File: JS_HW_1/js_hw_1_task_2.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//  Решить квадратные уравнения. Переменные называть по правилам.
+// Вывести в консоль ответы в виде "Ответ к уравнению 1: <корень>", "Ответ к уравнению 2: <корень> и <корень>"
+// 1.  x2 - 6x + 9 = 0. - один корень
+// 2.  x2 - 4x - 5 = 0. - два корня
+
+let a1 = 1;
```
Comment (anatoly-karpovich) [inline JS_HW_1/js_hw_1_task_2.js]: I believe almost all variables in your code should be created with const, not let) except variables that can be changed

File: JS_HW_1/js_hw_1_task_3.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+//  Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются
+
+function randomN(min, max) {
+    let n = min + Math.random() * (max + 1 - min);
+    return Math.floor(n);
+  };
+  console.log(
+    randomN(1, 9)
```
Comment (anatoly-karpovich) [inline JS_HW_1/js_hw_1_task_3.js]: As per requirements we must use only one number, but each call of randomN function returns new value. Need to store created one in a variable and use.

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: JS_HW_1/js_hw_1_task_3.js (hunk 2 above)
Comment (Labrosimova) [reply JS_HW_1/js_hw_1_task_3.js]: @anatoly-karpovich Thank you for your comments! I've updated the documents and added them to the branch.

**Review by anatoly-karpovich — APPROVED** [review]
Good job!

**Pull request merged**